src/dotnet/APIView/APIViewWeb/Client/src/reviews.ts

```diff
@@ -1,158 +1,104 @@
 ﻿$(() => {
   // Search
-  const languageSelect = $("#reviews-table-language-filter");
+  const reviewsFilterPartial = $( '#reviews-list-partial' );
+  const pkgNameFilter = $( '#name-filter-bootstraps-select' );
+  const languageFilter = $( '#language-filter-bootstraps-select' );
+  const tagFilter = $( '#tags-filter-bootstraps-select' );
+  const authorFilter = $( '#author-filter-bootstraps-select' );
+  const openFilter = $( '#open-filter-check' );
+  const closedFilter = $( '#closed-filter-check' );
+  const manualFilter = $( '#manual-filter-check' );
+  const automaticFilter = $( '#automatic-filter-check' );
+  const pullRequestFilter = $( '#pullrequest-filter-check' );
+  const approvedFilter = $( '#approved-filter-check' );
+  const pendingFilter = $( '#pending-filter-check' );
+
   const searchBox = $('#reviews-table-search-box');
   const searchContext = $('.review-name') as any;
-  const serviceGroupRows = $('.service-group-row');
-  const packageGroupRows = $('.package-group-row');
-  const reviewHeaderRows = $('.review-rows-header');
-  const packageDataRows = $('.package-data-row');
 
   // Enable tooltip
   (<any>$('[data-toggle="tooltip"]')).tooltip();
 
-  function toggleServiceGroup(serviceRow, state) {
-    var serviceGroupTag = serviceRow[0].id;
-    var serviceRowIcon = serviceRow.find('i').first();
-    var packageGroupRow = $(`.${serviceGroupTag}`);
-    if (state == "closed")
-    {
-      packageGroupRow.removeClass('hidden-row');
-      serviceRowIcon.removeClass('fa-angle-right').addClass('fa-angle-down');
-      serviceRow.addClass('shadow-sm');
-    }
-    else
-    {
-      packageGroupRow.addClass('hidden-row');
-      serviceRowIcon.removeClass('fa-angle-down').addClass('fa-angle-right');
-      serviceRow.removeClass('shadow-sm');
-    }
+  function updateListedReviews(pageNo=0)
+  {
+    var uri = "?handler=reviewspartial";
+
+    pkgNameFilter.children(":selected").each(function() {
+      uri = uri + '&packageNames=' + encodeURIComponent(`${$(this).val()}`);
+    });
+
+    languageFilter.children(":selected").each(function() {
+      uri = uri + '&languages=' + encodeURIComponent(`${$(this).val()}`);
+    });
+
+    tagFilter.children(":selected").each(function() {
+      uri = uri + '&tags=' + encodeURIComponent(`${$(this).val()}`);
+    });
+    
+    authorFilter.children(":selected").each(function() {
+      uri = uri + '&authors=' + encodeURIComponent(`${$(this).val()}`);
+    });
+
+    uri = uri + '&isOpen=' + openFilter.prop('checked');
+    uri = uri + '&isClosed=' + closedFilter.prop('checked');
+    uri = uri + '&isManual=' + manualFilter.prop('checked');
+    uri = uri + '&isAutomatic=' + automaticFilter.prop('checked');
+    uri = uri + '&isPullRequest=' + pullRequestFilter.prop('checked');
+    uri = uri + '&isApproved=' + approvedFilter.prop('checked');
+    uri = uri + '&isPending=' + pendingFilter.prop('checked');
+    uri = uri + '&offset=' + (pageNo * 50); //50 is the default size of a page
```

Comment:
I testes with 200 and I feel like it will make the page refresh lag. 50 seems to be a healthy size. In subsequent changes I will make the page size changeable form the UI.